snapcraft.yaml

```diff
@@ -0,0 +1,41 @@
+name: subiquity
+version: "0.0.32"
+summary: Ubuntu installer
+description: The Ubuntu server installer
+confinement: classic
+
+apps:
+  subiquity:
+    command: usr/bin/subiquity
+  console-conf:
+    command: usr/bin/console-conf
+  probert:
+    command: bin/probert
+
+parts:
+  subiquity:
+    plugin: python
+    build-packages: [python-setuptools]
+    stage-packages: [curtin]
+    python-packages:
+      - urwid
+      - pyyaml
+      - pyudev
+      - attrs
+    source: https://github.com/CanonicalLtd/subiquity.git
```

Comment:
It is if you're not ready to take the steps, and as above, this is only while this is not set up to do daily builds. This avoids cases where changes are not pushed and forgotten about.